raspbian/applications/vcgencmd.md

```diff
@@ -0,0 +1,133 @@
+
+The `vcos` cammand has a number of sub commands
+
+`version` Displays the build date and version of the firmware on the VideoCore.
+`log status` Displays the error log status of the various VideoCore software areas.
+
+#### version
+
+Displays the build date and version of the firmware on the VideoCore.
+
+#### get_camera
+
+Displays the enabled and detected state of the official camera. 1 means yes, 0 means no. Whilst all firmware (except cutdown versions) will support the camera, this support needs to be enabled by using [raspi-config](../../configuration/raspi-config.md).
+
+#### get_throttled
+
+Returns the throttled state of the system. This is a bit pattern.
+
+| Bit | Meaning |
+|:---:|---------|
+| 0   | Under-voltage detected |
+| 1   | Arm frequency capped |
+| 2   | Currently throttled |
+| 3   | Soft temperature limit active |
+| 16  | Under-voltage has occurred |
+| 17  | Arm frequency capped has occurred |
+| 18  | Throttling has occurred |
+| 19  | Soft temperature limit has occurred |
+
+#### measure_temp
+
+Returns the temperature of the SoC as measured by the on board temperature sensor
+
+#### measure_clock [clock]
+
+This returns the current frequency of the specified clock. The options are:
+
+| clock | Description |
+|:-----:|-------------|
+| arm   | ARM cores |
+| core  | VC4 scaler cores |
+| H264  | H264 block |
+| isp   | Image system Pipeline |
+| v3d   | 3D block |
+| uart  | UART |
+| pwm   | PWM block (analogue audio output) | 
+| emmc  | SD card interface |
+| pixel | Pixel valve |
+| vec | Analogue video encoder |
+| hdmi | HDMI |
+| dpi | Display Peripheral Interface |
+
+e.g. `vcgencmd measure_clock arm`
+
+#### measure_volts [block]
+
+Displays the current voltages used by the specific block.
+
+| block | Description |
+|:-----:|-------------|
+| core | VC4 core voltage |
+| sdram_c | |
+| sdram_i | |
+| sdram_p | |
+
+#### otp_dump
+
+Displays the content of the One Time Programmable (OTP) memory, which is part of the SoC. These are 32 bit values, indexed from 8 to 64. See the [OTP bits page](../../../hardware/raspberrypi/otpbits.md) for more details.
+
+#### get_mem
+
+Reports on the amount of memory allocated to the ARM cores `vcgencmd get_mem arm` and the VC4 `vcgencmd get_mem gpu`.
+
+**Note:** On a Raspberry Pi 4 with greater than 1GB of RAM, the `arm` option is inaccurate. This is because the GPU firmware which implements this command is only aware of the first GB of RAM on the system, so the `arm` setting will always return 1GB minus the `gpu` memory value. To get an accurate report of the amount of ARM memory, use one of the standard Linux commands, such as `free` or `cat /proc/meminfo`
+
+#### codec_enabled [type]
+
+Reports whether the specified CODEC type is enabled. Possible options for type are AGIF, FLAC, H263, H264, MJPA, MJPB, MJPG, **MPG2**, MPG4, MVC0, PCM, THRA, VORB, VP6, VP8, **WMV9**, **WVC1**. Those highlighted currently require a paid for licence. 
+
+#### get_config [type]
+
```

Comment:
Note that the name of a configuration item can be given in place of the type.

Reply:
And the parameter isn't optional (if we're following the `[]` convention).

Reply:
reworded slightly.